jax/_src/pallas/mosaic_gpu/pallas_call_registration.py

```diff
@@ -42,6 +42,7 @@ def pallas_call_lowering(
     compiler_params: dict[str, Any],
 ):
   if interpret:
+    # TODO(necula): is this still needed?
```

Comment:
Nope, I think you can safely remove it everywhere.

Reply:
I will do that in a separate cleanup PR that I am preparing, with this and other refactoring.